src/iap.ts

```diff
@@ -108,14 +110,14 @@ export const initConnection = (): Promise<boolean> =>
 
       return myRNIapModule.initConnection();
     },
-  })();
+  }) || Promise.resolve)();
```

Comment:
Don't you think we should reject it here instead of resolving it? Like below.
```
Promise.reject('Current platform is not supported in react-native-iap');
```

Also, same as other changes.

Reply:
Hello @hyochan , thank you for looking at this PR. The reason I went with resolve and not reject was because it's consistent with the other responses in that method here: ```if (!RNIapIos) return Promise.resolve();``` and here: ```if (!RNIapModule || !RNIapAmazonModule) return Promise.resolve();```